cmd/s3-gw/app.go

```diff
@@ -170,7 +170,7 @@ func newApp(ctx context.Context, l *zap.Logger, v *viper.Viper) *App {
 	ctr = auth.New(neofs.NewAuthmateNeoFS(conns), key, getAccessBoxCacheConfig(v, l))
 	handlerOptions := getHandlerOptions(v, l)
 
-	if caller, err = handler.New(l, obj, handlerOptions); err != nil {
+	if caller, err = handler.New(l, obj, nc, handlerOptions); err != nil {
```

Comment:
Shouldn't we remove `nc` from `App` struct (app.go:184)?

Reply:
Yes, unused. I will do it in separate commit after merge.